data_source_ssh_tunnel.go

```diff
@@ -26,10 +27,17 @@ func dataSourceSSHTunnel() *schema.Resource {
 				Required:    true,
 				Description: "The hostname",
 			},
+			"ssh_agent": &schema.Schema{
+				Type:        schema.TypeBool,
+				Optional:    true,
+				Description: "use ssh agent",
+				Default:     false,
```

Comment:
I feel like it might make sense to default this to `true`, and then document that the SSH agent will be automatically used if `SSH_AUTH_SOCK` is set.

Reply:
Agree, we'll evaluate the SSH_AUTH_SOCK value.

Reply:
Added a comment to the TF example.